Reference/Templating/Modelsbuilder/Introduction.md

```diff
@@ -32,7 +32,7 @@ And once you are using strongly typed models, there are some [cool things](CoolT
 
 ## Installing
 
-Starting with version 7.4.0, Umbraco.ModelsBuilder is bundled with the main Umbraco distribution. It is, however, also available as a NuGet package. See the [https://github.com/zpqrtbnk/Zbu.ModelsBuilder/releases](releases) page for more details.
+Starting with version 7.4.0, Umbraco.ModelsBuilder is bundled with the main Umbraco distribution. It is, however, also available as a NuGet package. See the [releases](https://github.com/zpqrtbnk/Zbu.ModelsBuilder/releases) page for more details.
```

Comment:
As we are referencing to an external website outside the documentation, I would mention this! 
Otherwise people would expect this is an internal link.